packages/user-group-service/src/helpers.ts

```diff
@@ -1,9 +1,11 @@
 import { createClient } from 'ldapjs';
 import { Users } from './users/schema';
-import * as _ from 'lodash';
+const _ = require("lodash");
```

Comment:
Can we use the old import model or just use the method that you need to be used ```import  { isEmpty } from 'lodash';```.

Reply:
I would not recommend using `require` over es6 `import`.
But +1 to importing the function itself, although it doesn't affect the final build too much.